pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -25,6 +27,19 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/resolver/projection"
 )
 
+const (
+	// TODO: Move to operator-framework/api/pkg/operators/v1alpha1
+	// BundleLookupFailed describes conditions types for when BundleLookups fail
+	BundleLookupFailed operatorsv1alpha1.BundleLookupConditionType = "BundleLookupFailed"
+
+	// TODO: This can be a spec field
+	// BundleUnpackTimeoutAnnotationKey allows setting a bundle unpack timeout per InstallPlan
+	// and overrides the default specified by the --bundle-unpack-timeout flag
+	// The time duration should be in the same format as accepted by time.ParseDuration()
+	// e.g 1m30s
+	BundleUnpackTimeoutAnnotationKey = "bundle-unpack-timeout"
```

Comment:
This was largely motivated by the e2e test's need to set a shorter timeout without reconfiguring the catalog operator deployment flags.

And I see your point but @benluddy and I thought this might be more easily back-portable than a proper spec field for now.

Reply:
ah, the e2e test angle makes sense as a tradeoff.